lib/brain.js

```
@@ -3115,6 +3129,10 @@ Brain.prototype.updateTx = function updateTx (updateTx) {
       this.tx = tx
       return tx
     })
+    .catch(e => {
+      console.log(e)
+      return newTx
+    })
```

I'm unsure about this one. There are six functions that call updateTx and only one of them doesn't have proper error handling, with this we'll be masking the error for everyone else.

Also that one usage is in handleScan, at which point there are no bills in the system, so it's better to just fail gracefully over there.

---

Makes sense, done.